iptables/iptables.go

```diff
@@ -304,15 +304,21 @@ func Exists(table Table, chain string, rule ...string) bool {
 
 	// try -C
 	// if exit status is 0 then return true, the rule exists
-	if _, err := Raw(append([]string{
-		"-t", string(table), "-C", chain}, rule...)...); err == nil {
+	_, err := Raw(append([]string{"-t", string(table), "-C", chain}, rule...)...)
+	if err == nil {
 		return true
 	}
+	if !strings.Contains(err.Error(), "unknown option") {
```

Comment:
This is the output of unsupported -C option of iptables v1.4.7. It doesn't contain "unknown option"
```
$ iptables -t nat -C PREROUTING -m addrtype --dst-type LOCAL -j DOCKER
iptables v1.4.7: option `-C' requires an argument
Try `iptables -h' or 'iptables --help' for more information.
```